scripts/run_external_device.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters"
+    exit 1
+fi
+
+export WORKSPACE=${WORKSPACE:-$(pwd)}
+export SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -f "$SCRIPTS_DIR/env/$1.sh" ]; then
```

Comment:
It would be nice to create a variable with a meaningful name instead of using `$1`